spec/bolt/node/winrm_spec.rb

```diff
@@ -161,7 +161,7 @@ def stub_winrm_to_raise(klass, message)
     end
   end
 
-  it "does not reuse host for multiple PowerShell scripts", winrm: true do
```

Comment:
This 2nd commit shows the behavioral differences around state between `powershell.exe` for every script, and invoking inside the current session.

Environment variables and globally scoped PowerShell variables remain accessible to additional invocations against the same host.

Were we to stop writing the PowerShell scripts to files, and instead invoke them in the current shell, the scoping rules would change yet again.

I figure these tests provide a good initial place to document what state is accessible later and why...